### PR TITLE
Update README.md

### LINKED ISSUE #1847: [BUG] Incomplete welcome readme
**Describe the bug**

The welcome section in the main Readme does not make sense. It currently reads 

> OpenSearch Dashboards is an open source search and analytics visualization.

**To Reproduce**
1. Go to [`./Readme.md`](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/README.md?plain=1#L11)
4. See text

**Expected behavior**
Expect the welcome text to be clearer about what the project is/does

### PR DESCRIPTION
**Pull request by vagimeli**
### Description

Updates Welcome section

### Issues Resolved

Fixes #1847

### Check List

- [ ] All tests pass
  - [ ] `yarn test:jest`
  - [ ] `yarn test:jest_integration`
  - [ ] `yarn test:ftr`
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [X] Update [CHANGELOG.md](./../CHANGELOG.md)
- [X] Commits are signed per the DCO using --signoff

### CODE REVIEW

**Comment by joshuarrrr** [comment]
@vagimeli The DCO check is failing because the commit is signed with your hidden GitHub email address. Is your GitHub email set to public? See https://github.com/opensearch-project/.github/blob/main/CONTRIBUTING.md#troubleshooting-failed-dco-checks

**Comment by vagimeli** [comment]
> @vagimeli The DCO check is failing because the commit is signed with your hidden GitHub email address. Is your GitHub email set to public? See https://github.com/opensearch-project/.github/blob/main/CONTRIBUTING.md#troubleshooting-failed-dco-checks

@joshuarrrr I've addressed the private v. public issue. Will DCO pass it now?

**Comment by AMoo-Miki** [comment]
Melissa, looks like the DCO check is still not able to access your GitHub profile to verify the email belongs to you. However, as your commits were signed, and I know for a fact that the changes in the commits are your own, I will pass the DCO.

File: README.md [hunk 1]
```diff
@@ -8,9 +8,13 @@
...
+OpenSearch Dashboards is an open-source data visualization tool designed to work with OpenSearch. OpenSearch Dashboards gives you data visualization tools to improve and automate business intelligence and support data-driven decision-making and strategic planning.
 
-Feel free to take a look at what the community has been up to, and then head over to the [Project Board](https://github.com/opensearch-project/OpenSearch-Dashboards/projects) to track release targets, or jump in and [start opening issues](https://github.com/opensearch-project/OpenSearch-Dashboards/issues/new/choose), [set up your development environment](DEVELOPER_GUIDE.md#getting-started), or [start contributing](CONTRIBUTING.md).
+We aim to be an exceptional community-driven platform and to foster open participation and collective contribution with all contributors. Stay up to date on the what's happening with the OpenSearch Project by tracking GitHub [issues](https://github.com/opensearch-project/OpenSearch-Dashboards/issues) and [pull requests](https://github.com/opensearch-project/OpenSearch-Dashboards/pulls).
```
Comment (AMoo-Miki) [inline README.md:13]: This sounded wired to me. How would you feel about these alternates? (or am I way off)

```suggestion
We aim to be an exceptional community-driven platform and to foster open participation and collective contribution with all contributors. Stay up to date on what's happening with the OpenSearch Project by tracking GitHub [issues](https://github.com/opensearch-project/OpenSearch-Dashboards/issues) and [pull requests](https://github.com/opensearch-project/OpenSearch-Dashboards/pulls). 
```

OR

```suggestion
We aim to be an exceptional community-driven platform and to foster open participation and collective contribution with all contributors. Stay up to date with what's happening at the OpenSearch Project by tracking GitHub [issues](https://github.com/opensearch-project/OpenSearch-Dashboards/issues) and [pull requests](https://github.com/opensearch-project/OpenSearch-Dashboards/pulls). 
```

**Comment by vagimeli** [comment]
> Melissa, looks like the DCO check is still not able to access your GitHub profile to verify the email belongs to you. However, as your commits were signed, and I know for a fact that the changes in the commits are your own, I will pass the DCO.
> 
> PS, I also updated the PR text to indicate that it fixes #1847

@AMoo-Miki Thanks for passing the DCO. I resolved the issue with my email being public. I also accepted your changes. Thanks for catching those typos :)

**Review by AMoo-Miki — APPROVED** [review]

**Review by ashwin-pc — APPROVED** [review]

**Pull request merged**